Pixhawk 6C & X500 v2 developer guides (#2121)
PX4/PX4-user_guide: en/frames_multicopter/holybro_x500v2_pixhawk6c.md

> # Holybro X500 V2 + Pixhawk 6C (PX4 Dev Kit)

Reviewer: We can't just call it the PX4 Dev Kit - this is already the second "Devkit" produced by Holybro, and they name it the "Holybro X500 V2 ARF Kit" at their link.

This topic provides full instructions for building the [Holybro X500 V2 ARF Kit](http://shop.holybro.com/x500-v2-kit_p1288.html), also known as the Holybro PX4 Dev Kit.

> ![The fully built vehicle with props removed](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/kit_no_props.jpg)

Reviewer: FYI I shrunk all the images by file size. This from 6Mb to about 300Kb. Not completely lossless, but you won't notice it on a normal computer screen.
I also used the standard markdown reference for images - this is better because it allows the images to be zoomable by clicking them.

> 

Reviewer: [Previous version](http://docs.px4.io/main/en/frames_multicopter/holybro_x500V2_pixhawk5x.html) had a lot more text here, including introduction Bill of materials, key information, hardware, electronics. Do you plan on adding that?

Reviewer: As discussed these are useful. Why? Because they allow users to check they have everything needed before they start. They also identify any bits you have to purchase that may no longer be in the kit.

That said, right now they are not essential, and they are less essential for something like this which is delivered as a kit by a reliable vendor.

## Assembly

> ### Holders

Reviewer: Note, I changed all the headings to be "First letter of each word capitalized". This is common across the whole document set. Please follow the patterns you see elsewhere.

> ### Holders

Reviewer: What is a Holder?

Author: > What is a Holder?

That would be the payload and battery holder assembly.

Reviewer: I've updated title to reflect that

> **Screw**-  Sunk Screw M2.5*6 12pcs

Reviewer: What does this mean? Is this the stuff you need to do this section? Aren't there other things you need here?

Author: > What does this mean? Is this the stuff you need to do this section? Aren't there other things you need here?

That's at the beginning of every section to mention which screws would be required ahead.

1. Insert the hanger rubber ring gasket in each of their respective hangers.
   Do not use sharp objects to press the rubbers inside.

>    @[youtube](https://youtu.be/4Tid-FCP_aI)

Reviewer: FYI, you were using bullets. If the steps are in order, best to use numbers. Two things to note here:
1. You can always number using `1.`. Markdown will auto number for you.
2. If you indent to match the number, then the contents are part of the numbered item. So you get a nice indent on rendering.

1. Take the battery mounting board and screw it with the slide bar clip using the Sunk Screw M2.5*6.

>    @[youtube](https://youtu.be/9E-rld6tPWQ)

Reviewer: For every image, please replace these with a "final" image that you can click to go to the youtube. 

The syntax will end up something like this:
```
[![](this_is_the_image_in_assets_directory)](https://youtu.be/9E-rld6tPWQ))
```

Don't get me wrong, I LOVE the videos. But in plenty of places they will be slow to load, if they are able to load at all. So what you want here is an image that shows the final state you want people to get to - in other words, if they don't see the video they can STILL use the document. 

Make sense? The images should be in the assets directory.

Reviewer: PS, here is a preview of how the links would behave: https://hamishwillee.github.io/multidocsify_test/#/redir_repo/en/akshata-01/PX4-user_guide/dev_guides/frames_multicopter/holybro_x500v2_pixhawk6c?id=power-module

Though note, the images are autogenerated in this case - I don't want to do that because the images are the initial image in the file, not the "final thing you want to produce"

Author: Got it, makes sense.

Author: @hamishwillee  
Since I do not have images, does a screenshot of the final image of each video work? 

Should I crop the part where YouTube's play & pause is seen? 
![image](https://user-images.githubusercontent.com/108534583/199400157-c9885953-9d59-4144-acbe-0a291bb39e20.png)

1. Screw 4 hangers to the Platform Board using Sunk Screw M2.5*6.

   @[youtube](https://youtu.be/4qIBABc9KsY)

1. Take the slide bar and insert 4 hangers to screw to the bottom plate later.

   @[youtube](https://youtu.be/CFx6Ct7FCIc)

1.  Now insert the battery holder and payload holders assembled in step 2 & 3

### Power Module

> 

Reviewer: It might be nice to say something about the power module here - what's it for? What limits does it have (e.g. what batteries can it be used with).

> **Screw**- Socket Cap Screw M2.5*6 8pcs | Locknut M3 4pcs |Nylon Standoff M3*5 4pcs | Screw M3*14 4pcs

Reviewer: Again, what's this? Why just screws. What else do you need for this?

1. Take the bottom plate and insert 4 M3*14 screws and fasten the nylon standoffs on the same.

   @[youtube](https://youtu.be/0knU3Q_opEo)

1. Place the Power distribution board and use the locknuts to assemble them. The power module PM02 (for Pixhawk 6C) would power this board

   @[youtube](https://youtu.be/IfsMXTr3Uy4)

1. Use Socket Cap Screws M2.5*6 and screw the bottom plate on the 4 hangers (that we inserted in the 2 bars on the 3rd step of HOLDER assembly)

   @[youtube](https://youtu.be/Qjs6pqarRIY)

### Landing Gear

@[youtube](https://youtu.be/mU4vm4zyjcY)

> @[youtube](https://youtu.be/7REaF3YAqLg)

Reviewer: The images don't match to numbers. Can you apply numbers and indentation so that this works as a "flow of actions"

1. To assemble the landing gear, loosen the pre-assembled screws of the Landing Gear-Cross Bar and insert the Landing Gear-Vertical Pole and fasten the same.

   @[youtube](https://youtu.be/zwa2L_daam8)

>    @[youtube](https://youtu.be/iDxzWeyCN54)

Reviewer: Ditto - is the intent that these are both under one number? Also for next item

1. Use the Socket Cap Screw M3*8 to screw the landing gears to the bottom plate

   @[youtube](https://youtu.be/3fNJQraCJx0)

   @[youtube](https://youtu.be/3en4DlQF4XU)

> *Because it’s cumbersome to insert the wires once the top plate is assembled, do the wiring beforehand. 

Reviewer: If this is a note, can you do it like a note using `:::note` syntax?

Although the design is well built such that you can do this later as well.
 
### Power

> 

Reviewer: Before you had "Power module" and now you have "Power". Can they merge? This feels very out of place.

Reviewer: It is, but I have round out of energy to fix. Can be done as a post process - the information is still here an useful.

- The Pixhawk 6C gets powered by a power module PM02 (in this case).
- This power module is supplied by a battery (4S 16.8V 5200 mAh)
- The motors get power through the power distribution board.

![motors_pdb_pixhawk6c](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/motors_pdb_pixhawk6c.png)

### Arms

> **Screw-** Socket Cap Screw M3*38 16pcs | Flange Locknut M3 16pcs

Reviewer: As above, what is this.

@[youtube](https://youtu.be/66Hfy6ysOpg)

1. Putting the arms is quite simple as the motors come pre-assembled.  - Ensure that you have the right numbered arm with its motor on the respective side.

   @[youtube](https://youtu.be/45KCey3WiJ4)

   :::tip
   Use your allen keys/ any elongated item and insert it on the opposite side of the bolt that you're trying to fasten.
   :::

1. Take one arm and insert the rectangle extrusion inside the rectangular hollow on the bottom plate.

   @[youtube](https://youtu.be/GOTqmjq9_3s)

1. While inserting the top plate on top of this the 3 piece assembly (bottom plate, top plate & arms) have to screwed using Socket Cap Screw M3*38 and Flange Locknut M3. 
1. Hold one side using the mini cross wrench provided in the developer kit.

   @[youtube](https://youtu.be/2rcNVekJQd0)

1. Do not fasten any screws before all 3 motors are in place as this might make it difficult while you’re assembling the 3rd and 4th motor.

   @[youtube](https://youtu.be/SlKRuNoE_AY)

### Propellers

> @[youtube](https://youtu.be/V7JXI5-qrAs)

Reviewer: This video is called propellors, but shows not a single propeller.

- The bottom plate indicates the direction of the motor.
- The propellers that have a white/silver coating go on that respective motor with the similar coat.
- The unlocking and locking of the propeller is indicated on the propeller itself.

> - Use the 4 propellers and insert them on the motors keeping the above 3 points in mind.

Reviewer: This is good information. It doesn't hurt to introduce the video and the text though. It just feels very abrupt - a video, then these points.

You could for example prefix this list with: "When attaching the propellers, note that:"

> *The below parts can be placed as per user

Reviewer: If it is a note, make into a note.

Reviewer: Also, I don't know what "below parts" you are talking about, or what placing as per user means.

### GPS

**Screw-** Locknut M3 4 pcs | Screw M3*10 4pcs

@[youtube](https://youtu.be/aiFxVJFjlos)

- Assemble the GPS by following the video.
- This guide uses the GPS mount location according to the suggestion in Holybro’s guide.
- Screw the GPS mount’s bottom end on the payload holder side using Locknut M3 & Screw M3*10

@[youtube](https://youtu.be/uG5UKy3FrIc)

### Pixhawk 6C

- The wire from the PM02 goes to POWER1 in Pixhawk
- The telemetry goes to TELEM1
- The GPS to GPS1

@[youtube](https://youtu.be/wFlr_I3jERQ)

### Companion Computer (Optional)

**Screw-** Socket Cap Screw M2.5*12 4pcs | Nylon Standoff M2.5*5 4pcs Locknut M2.5 4pcs

> - The X500 kit is awesome because of it’s design consideration of most common hardware used on drones.

Reviewer: This is confusing English and has nothing to do with Companion computers.

Reviewer: @akshata-01 I have no idea what this means. I'll make something up, but you will need to create a new PR to fix it after this merges.

> - The companion computer such as Raspberry Pi or Jetson nano can be placed here.

Reviewer: "Can be placed here". Where is "here"?

- Insert 4 Socket Cap Screw M2.5*12 and put the standoffs on the same.
- Now place the companion computer and assemble it using Locknut M2.5

### Camera

- Cameras such as Intel Realsense depth/ tracking camera or Structure Core can be mounted using the Depth Camera Mount
- Simply insert the mount inside the 2 bars and use the screws according to the camera you’re using.

> ![payloads_x500v2](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/payloads_x500v2.png)

Reviewer: I like this image?

> 

Reviewer: Missing:
- Installing PX4
- Tuning (might not be needed)
- Configuring.

- The unlocking and locking of the propeller is indicated on the propeller itself.
- Use the 4 propellers and insert them on the motors keeping the above 3 points in mind.

The following parts can be placed as per usual.

### GPS

**Screw-** Locknut M3 4 pcs | Screw M3*10 4pcs

1. Assemble the GPS by following the video.

   [![Assembly20](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/assembly20.png)](https://youtu.be/aiFxVJFjlos)
  
   This guide uses the GPS mount location suggested in Holybro’s guide.
1. Screw the GPS mount’s bottom end on the payload holder side using Locknut M3 & Screw M3*10

   [![Assembly21](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/assembly21.png)](https://youtu.be/uG5UKy3FrIc)

### Pixhawk 6C

- The wire from the PM02 goes to POWER1 in Pixhawk
- The telemetry goes to TELEM1
- The GPS to GPS1

[![Assembly22](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/assembly22.png)](https://youtu.be/wFlr_I3jERQ)

### Companion Computer (Optional)

**Screw-** Socket Cap Screw M2.5*12 4pcs | Nylon Standoff M2.5*5 4pcs Locknut M2.5 4pcs

> The X500 kit is provides space for a companion computer, such as Raspberry Pi or Jetson nano can be placed here [TBD].

Reviewer: @akshata-01 So I am expecting an image or something later

- Insert 4 Socket Cap Screw M2.5*12 and put the standoffs on the same.
- Now place the companion computer and assemble it using Locknut M2.5

### Camera

- Cameras such as Intel Realsense depth/ tracking camera or Structure Core can be mounted using the Depth Camera Mount
- Simply insert the mount inside the 2 bars and use the screws according to the camera you’re using.

![payloads_x500v2](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/payloads_x500v2.png)

> ## Install/Configure PX4

Reviewer: I made this up based on the previous guide. However it will only be accurate for v1.13. It is not accurate for main.

- The unlocking and locking of the propeller is indicated on the propeller itself.
- Use the 4 propellers and insert them on the motors keeping the above 3 points in mind.

The following parts can be placed as per usual.

### GPS

**Screw-** Locknut M3 4 pcs | Screw M3*10 4pcs

1. Assemble the GPS by following the video.

   [![Assembly20](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/assembly20.png)](https://youtu.be/aiFxVJFjlos)
  
   This guide uses the GPS mount location suggested in Holybro’s guide.
1. Screw the GPS mount’s bottom end on the payload holder side using Locknut M3 & Screw M3*10

   [![Assembly21](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/assembly21.png)](https://youtu.be/uG5UKy3FrIc)

### Pixhawk 6C

- The wire from the PM02 goes to POWER1 in Pixhawk
- The telemetry goes to TELEM1
- The GPS to GPS1

[![Assembly22](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/assembly22.png)](https://youtu.be/wFlr_I3jERQ)

### Companion Computer (Optional)

**Screw-** Socket Cap Screw M2.5*12 4pcs | Nylon Standoff M2.5*5 4pcs Locknut M2.5 4pcs

> - The X500 kit is awesome because of it’s design consideration of most common hardware used on drones.
> - The companion computer such as Raspberry Pi or Jetson nano can be placed here.

Reviewer: ```suggestion
The X500 kit is provides space for a companion computer, such as Raspberry Pi or Jetson nano can be placed here [TBD].
```
